src/patternfly/components/Select/examples/index.js

```diff
@@ -61,6 +64,13 @@ export default (props) => {
       <Example heading="Select - single (expanded)" handlebars={SelectSingleExpandedExampleRaw} minHeight="20em">
         {selectSingleExpandedExample}
       </Example>
+      <Example
+        heading="Select - single direction top (expanded)"
```

Comment:
What do you think about renaming this to `Select - single (top expanded)`? "single direction top" reads like "direction" is part of "single," as in "single direction," instead of "direction top".